app/src/main/java/io/gnosis/safe/ui/dialogs/EnsInputDialog.kt

```diff
@@ -22,11 +24,12 @@ import kotlinx.coroutines.flow.asFlow
 import kotlinx.coroutines.flow.collect
 import kotlinx.coroutines.launch
 import pm.gnosis.model.Solidity
+import pm.gnosis.svalinn.common.utils.getColorCompat
 import pm.gnosis.svalinn.common.utils.showKeyboardForView
 import pm.gnosis.svalinn.common.utils.visible
 import javax.inject.Inject
 
-class EnsInputDialog : BaseViewBindingDialogFragment<DialogEnsInputBinding>() {
+class EnsInputDialog(private val selectedChain: Chain) : BaseViewBindingDialogFragment<DialogEnsInputBinding>() {
```

Comment:
selectedChain should be passed as an argument in a bundle

Reply:
https://developer.android.com/reference/android/app/Fragment.html#public-constructors_1